alert-policies/activemq/MessageEnqueueRate.yml

```diff
@@ -0,0 +1,27 @@
+name: Message Eequeue Rate
+
+description: |+
+  This alert is triggered when the Messages sent is less than 1 during 5 minutes.
```

Comment:
OLD:
This alert is triggered when the Messages sent is less than 1 during 5 minutes.
Updated need to:
This alert is triggered when the messages sent are less than 1 during 5 minutes.

Reply:
Hi @pkosanam,
Updated grammar mistake changes.